server/src/main/proto/spine_examples/kanban/rejections.proto

```diff
@@ -44,6 +44,16 @@ option java_multiple_files = false;
 import "spine_examples/kanban/identifiers.proto";
 import "spine_examples/kanban/wip_limit.proto";
 
+// A column cannot be added as its name is taken.
+message ColumnNameIsTaken {
```

Comment:
Let's say `...AlreadyTaken`. 

Right now it sounds a bit like an event. And we want the name to clearly speak of an erroneous outcome.